### ISSUE TITLE
Default values when call endpoint /me with custom User Model

### ISSUE THREAD

**Issue by MariusMez** [frankie567/fastapi-users#83]
Hi Frankie567
First of all thanks for this promising plugin ! 

I have added new fields to the User Model class like mentioned in documentation:

```
class User(BaseUser):
    fullname: Optional[str] = None
    creation_date: Optional[datetime] = datetime.utcnow()
```

When I call the register endpoint, the User is correctly saved with all fields completed.

_fullname = 'test'
creation_date = '2020-01-02T08:23:34.014678"_

But when after login I call the endpoint `/me`, only the 'base' fields (email, password, is_active) are set with the database value, the new fields (fullname, creation_date) are shown with their default value.

`{"id":"8ce5f915-3218-41b0-a75f-a18055b11176","email":"test@plop.fr","is_active":true,"is_superuser":true,"fullname":null,"creation_date":"2020-01-03T09:36:27.023413"}`

So maybe I'm missing a point or I need to surcharge other methods in my User Model class ?

**Comment by frankie567** [comment]
Hi @MariusMez!

Thank you for your feedback 🙂 I think I see what's going on here:

https://github.com/frankie567/fastapi-users/blob/c903b301615c6fd0e9bfe2c25cd3625fe52fc21e/fastapi_users/db/mongodb.py#L28

The DB adapters use the base Pydantic class ; thus they will discard the extra fields. I'll find a fix!

**Comment by MariusMez** [comment]
Cool !

I haven't test yet but maybe the 'extra' parameter can help: https://pydantic-docs.helpmanual.io/usage/model_config/

**Comment by frankie567** [comment]
@MariusMez Issue should now be fixed in [release 0.5.0](https://github.com/frankie567/fastapi-users/releases/tag/v0.5.0), at the price of some breaking changes.

If you have any questions, I would be happy to help 😄

**Comment by MariusMez** [comment]
@frankie567 Perfect, thank you 😊
